imports/methods/announcements/post.ts

```diff
@@ -0,0 +1,38 @@
+import { check } from 'meteor/check';
+import { ValidatedMethod } from 'meteor/mdg:validated-method';
+import { Meteor } from 'meteor/meteor';
+import Ansible from '../../Ansible';
+import Announcements from '../../lib/models/Announcements';
+import MeteorUsers from '../../lib/models/MeteorUsers';
+import PendingAnnouncements from '../../lib/models/PendingAnnouncements';
+import { userMayAddAnnouncementToHunt } from '../../lib/permission_stubs';
+
+export default new ValidatedMethod({
+  name: 'Announcements.methods.post',
+
+  validate({ huntId, message }: { huntId: unknown, message: unknown }) {
```

Comment:
One thing that I liked about the previous approach was that by checking each positional argument (and having to have the number match to have your callback get triggered), it made it very hard for a malicious client to smuggle data into the methods consumed by the server, by virtue of the fact that if the server didn't declare the positional argument, then additional garbage would be dropped before reaching the body of the method call.

It seems we lose that a little under this approach, by virtue of the fact that an object can contain additional garbage.  It's mitigated by the fact that when we do object destructuring we also tend to ignore unknown fields, so long as we never ever use `...rest` in any of our method signatures.  Maybe we can lint for that?  Or as noted in the top-level comment, passing the types through all the way would eliminate this entirely.